<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuousDataForm</class>
 <widget class="QDialog" name="ContinuousDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>452</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Continuous Data</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/icons.qrc">
    <normaloff>:/images/meta.png</normaloff>:/images/meta.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_box_pre_post">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>pre / post</string>
   </property>
   <widget class="QLabel" name="grp_1_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>group 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="grp_2_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Verdana'; font-size:8pt; font-weight:600; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;group 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="g1_pre_post_table">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>21</y>
      <width>550</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <row>
     <property name="text">
      <string>pre</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>post</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>n</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mean</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>sd</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>se</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>var</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>pval</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>low</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>high</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="g2_pre_post_table">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>550</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <row>
     <property name="text">
      <string>pre</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>post</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>n</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mean</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>sd</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>se</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>var</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>pval</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>low</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>high</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>correlation:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="correlation_edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="simple_table">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>550</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="rowCount">
    <number>2</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <row>
    <property name="text">
     <string>group 1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>group 2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>n</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>mean</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>sd</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>se</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>var</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pval</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>low</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>high</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>α:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alpha_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>.05</string>
   </property>
  </widget>
  <widget class="QLabel" name="ci_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(95% confidence interval)</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContinuousDataForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContinuousDataForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
